### PR TITLE
python/luhn: mentor notes

### PR DESCRIPTION
**Pull request by jeffdparker**
Third attempt to submit mentoring notes for luhn.

### CODE REVIEW

File: tracks/python/exercises/luhn/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+## Problem and challenges
```
Comment (SleeplessByte) [inline tracks/python/exercises/luhn/mentoring.md]:
```suggestion
### Problem and challenges
```

**Comment by SleeplessByte** [comment]
Looks great @jeffdparker ! Pinging @exercism/website-copy-python

**Comment by jeffdparker** [comment]
I'm still a babe in the woods here.  The only option I see is to close the request.

**Comment by cmccandless** [comment]
@jeffdparker no worries, the onus is on maintainers here to merge once all reviewing is passed. I've reopened the PR, and will go ahead and merge since it has passed review. Thanks!

**Comment by jeffdparker** [comment]
Thanks for your help!

**Comment by jeffdparker** [comment]
Thanks for your help, but help me understand - what was the proper action for me to take?

You turned it back to me to approve - what do I do at that point?

**Comment by cmccandless** [comment]
Actually, I approved the changes, leaving it for @SleeplessByte or another maintainer who is more active in this repo to merge.

> what was the proper action for me to take?

Nothing, actually. Unless more changes were requested of you, you get to sit back and wait for the merge to be done!

**Pull request merged**